### DISCUSSION TITLE
Adding Berries & Plums to Entry Level Gathering

### DISCUSSION THREAD

**Discussion by jasper-one** [EveronLife/EveronLife#49] [Ideas]
**Summarize your proposal**
The entry-level gathering is off to a great start with the classic apple picking. As a first-time modder, I would love to try and expand on this by adding some other entry-level gathering items that also leverage existing game assets. 

**Describe the solution you'd like**
I would like to have a go at implementing the two other trees that have fruits. 
t_sorubs_aucuparia_2f.et:
![image](https://user-images.githubusercontent.com/106525617/170971333-a3edc73b-8ae1-4d76-89e9-85fa49d71eb7.png)

t_prunus_dom_estica_2s.et:
![image](https://user-images.githubusercontent.com/106525617/170971192-06bef9fb-8e45-463e-9398-a4c159dc9f5f.png)

This would require the upload of a low poly model and small texture for both plums and the small red berries from the trees and then the configuration of the existing assets to make use of the same functionality as the apples do currently. Potentially I could also reskin the current apples with a plum texture to make use of the same model (given we really want to keep the mod file size down).

If possible, I would like to be assigned this personally to get the hang of contributing.

**Comment by jasper-one** [comment]
I have made a fork with Plum Gathering added, just not when / if it's a good idea to make a pull request. https://github.com/KasperOnPC/EveronLife

**Comment by SaxW** [comment]
Added berries.
https://github.com/EveronLife/EveronLife/pull/94
